e2e/cypress/integration/enterprise/guest_accounts/guest_removal_ui_spec.js

```diff
@@ -0,0 +1,99 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. #. Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+/**
+ * Note: This test requires Enterprise license to be uploaded
+ */
+import * as TIMEOUTS from '../../../fixtures/timeouts';
+
+let guest;
+let team1;
+let team2;
+
+function removeUserFromAllChannels(verifyAlert) {
+    // # Remove the Guest user from all channels of a team as a sysadmin
+    const channels = ['Town Square', 'Off-Topic'];
+    channels.forEach((channel, index) => {
+        // # Remove the Guest User from channel
+        cy.getCurrentChannelId().then((channelId) => {
+            cy.removeUserFromChannel(channelId, guest.id);
+        });
+
+        // * Verify if guest user gets a message when the channel is removed
+        if (index < channels.length - 1 || verifyAlert) {
```

Comment:
This feels a bit confusing to me. Does this modal only appear if it's NOT the last channel? Given that there are just two channels I'd be more inclined to do, albeit a bit nit picky perhaps:

```js
if (channel === 'Town Square' || verifyAlert) {
    // sweet code
}
```

If you don't feel strongly about this, I've approved the PR on my end.

Reply:
Hi @thekiiingbob 
Yes that's correct. I will simplify the code for better understanding as suggested.